extension/public/popup.js

```diff
@@ -101,10 +101,11 @@ const renderAllowedWords = () => {
         : "";
 
     for (let i = 0; i < numTagItems; i++) {
-      tagItems += `<div class="tag-item">${allowedWordsData[i]}</div>`;
+      tagItems += `<div class="tag-item" ><span class="tagText">${allowedWordsData[i]}</span></div>`;
```

Comment:
Can the `tagText` (or `tag-text`) go beside `tag-item` e.g. `class="tag-item tag-text`? This will reduce unnecessary complexity.

Reply:
If u mean can div class = "tag-item tag-text" then it would not work

Reply:
What about `class="tag-item tagText`?